src/Serilog.Sinks.EventLog/Sinks/EventLog/EventIdHash.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright 2016 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+
+namespace Serilog.Formatting.Compact
```

Comment:
Yeah, sorry, I saw the r# warning for this too and my brain thought "well i'll keep the namespace the same [as the other `EventIdHash`] for consistency.  Current me has no idea what past me meant by "consistency".